sections/appendix/problems.tex

```diff
@@ -47,8 +47,20 @@ \subsubsection{Unable to reverse engineer the \gls{lda} generative process}
 In the end we decided that reverse engineering the \gls{lda} process, was not an effective strategy, and went on to try getting more context to our queries using 'query extension'.
 
 
-\subsection{Duplicates within the dataset}
+\subsubsection{Duplicates within the dataset}
 Within Nordjyske's data, each article is published in several different magazines. 
 This results in the dataset consisting of many duplicates. 
 To combat this problem, we make a dictionary of articles, so that we only process unique articles.
 However when we have preprocessed the Nordjyske data, there are still some duplicates, since articles containing information like football scoreboards, contain different scores before preprocessing but the same information after preprocessing, since we remove numbers from the articles.
+
+
+\subsubsection{Problems with word value multiplication}
+We use \autoref{eq:query_prob} to calculate the probability of a query having been generated by a specific document. 
+However, because all word probabilities $P(w|d)$ are multiplied together, having a value of 0 just one word will make the whole query have a value of 0.
```

Comment:
```suggestion
However, because all word probabilities $P(w|d)$ are multiplied together, having a value of 0 for just one word will make the whole query have a value of 0.
```